.travis.yml

```diff
@@ -1,21 +1,164 @@
 language: python
-
-python:
-  - "3.5"
-  - "3.6"
-  - "3.7"
-  - "3.8"
-
-install:
-  - "pip install -r requirements.txt"
-
-script:
-  - "make pylint"
-  - "make flake8"
-  - "make unittest"
+dist: focal
+sudo: false
 
 addons:
   apt:
+    sources:
+      - sourceline: deb http://apt.postgresql.org/pub/repos/apt focal-pgdg main
+        key_url: https://www.postgresql.org/media/keys/ACCC4CF8.asc
     packages:
       - libsnappy-dev
-      - postgresql-9.4
+      - postgresql-9.6
+      - postgresql-10
+      - postgresql-11
+      - postgresql-12
+
+
+stages:
+  - lint
+  - unittest
+
+inc_before_install: &inc_before_install
+  # Setup the Postgres repositories
+  - sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+  - wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+  - sudo apt update
+
+inc_install: &inc_install
+  # Setup common python dependencies
+  - pip install -r requirements.txt
+  - pip install -r requirements.dev.txt
+  - pip install -e .
+
+inc_unittest: &inc_unittest
+  - make unittest
+
+jobs:
+  allow_failures:
+    # Don't require python to succeed
+    - python: nightly
```

Comment:
This seems to be failing currently and I don't know if it provides any value anyhow. I don't expect anyone to make fixes because this failed and if it will just be ignored we might as well save some time and resources by not running it.

Reply:
Agreed. Fixed.